bot/handlers/sending_videos/compile_clips_handler.py

```diff
@@ -60,79 +65,156 @@ async def _do_handle(self, message: Message) -> None:
+        if errors:
+            return await self._answer(message, "\n".join(errors))
 
         if not selected_segments:
             return await self.__reply_no_matching_segments_found(message)
 
-        if not await DatabaseManager.is_admin_or_moderator(message.from_user.id) and len(selected_segments) > settings.MAX_CLIPS_PER_COMPILATION:
-            await self._answer(message,get_max_clips_exceeded_message())
-            return
+        if (
+            not await DatabaseManager.is_admin_or_moderator(message.from_user.id)
+            and len(selected_segments) > settings.MAX_CLIPS_PER_COMPILATION
+        ):
+            return await self._answer(message, get_max_clips_exceeded_message())
 
         total_duration = 0
         for segment in selected_segments:
-            duration = (segment["end"] + settings.EXTEND_AFTER) - (segment["start"] - settings.EXTEND_BEFORE)
+            duration = (
+                (segment["end"] + settings.EXTEND_AFTER)
+                - (segment["start"] - settings.EXTEND_BEFORE)
+            )
             total_duration += duration
             await self._log_system_message(
-                logging.INFO, f"Selected clip: {segment['video_path']} "
+                logging.INFO,
+                f"Selected clip: {segment['video_path']} "
                 f"from {segment['start']} to {segment['end']} with duration {duration}",
             )
-            await self._log_system_message(logging.INFO, f"Total duration: {total_duration}")
 
-        if await self._handle_clip_duration_limit_exceeded(message, total_duration):
+        if await self._check_clip_duration_limit(message, total_duration):
             return
-        compiled_output = await ClipsCompiler.compile(message, selected_segments, self._logger)
 
+        compiled_output = await ClipsCompiler.compile(message, selected_segments, self._logger)
         await self._answer_video(message, compiled_output)
         await process_compiled_clip(message, compiled_output, ClipType.COMPILED)
 
-        await self._log_system_message(logging.INFO, get_compilation_success_message(message.from_user.username))
+        await self._log_system_message(
+            logging.INFO,
+            get_compilation_success_message(message.from_user.username),
+        )
 
+    # Parsing Methods
     @staticmethod
-    def __parse_segments(content: List[str], segments: List[Dict[str, Union[str, float]]]) -> List[Dict[str, Union[str, float]]]:
+    def __parse_segments(
+        content: List[str],
+        segments: List[Dict[str, Union[str, float]]],
+    ) -> (List[Dict[str, Union[str, float]]], List[str]):
         selected_segments = []
-        for index in content:
-            if index.lower() == "wszystko" or index.lower() == "all":
+        errors = []
+
+        # noinspection PyPep8Naming
+        ALL_KEYWORDS = {"all", "wszystko"}
+
+        for arg in content:
+            if arg.lower() in ALL_KEYWORDS:
```

Comment:
//optional ja bym tak to zrobil, nie trzeba sie pierdolic teraz z tymi pepami

```suggestion
        for arg in content:
            if arg.lower() in {"all", "wszystko"}:
```